### PR TITLE
build feature edit, create, delete project

### PR DESCRIPTION
**Pull request by tuanhai2712**

### CODE REVIEW

**Comment by TaHieu-scuti** [comment]
Pls run phpcs fixer again.

File: app/Member.php [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 class Member extends Model
 {
-    protected $fillable = 
-    ['member_name', 'member_info', 
-    'member_phone', 'member_dateofbirth', 
+    protected $fillable =
+    ['member_name', 'member_info',
```
Comment (TaHieu-scuti) [inline app/Member.php]: Indentation wrong. Pls run phpcs for this file.

File: app/Http/Controllers/ProjectController.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        ProjectValidatorRequest $request,
+        $id = null
+    ) {
+        $dataNewProject = $request->all();
+        $profileNewProject = new Project();
+
+        if ($id !== null) {
+            $profileNewProject = Project::find($id);
+        }
+            $profileNewProject->project_name = $dataNewProject['project_name'];
```
Comment (TaHieu-scuti) [inline app/Http/Controllers/ProjectController.php]: Indentation wrong. Pls run phpcs for this file.

File: app/Member.php [hunk 3]
```diff
@@ -6,9 +6,10 @@
 
 class Member extends Model
 {
-    protected $fillable = 
-    ['member_name', 'member_info', 
-    'member_phone', 'member_dateofbirth', 
-    'member_avatar', 'member_position', 'member_gender'
+    protected $fillable =
```
Comment (ngon12) [inline app/Member.php]: Please format like that.

```
protected $fillable = [
    'project_name', 'project_info', 
    'project_deadline', 'project_type', 
    'project_status'
];
```

File: app/Project.php [hunk 4]
```diff
@@ -7,4 +7,6 @@
 class Project extends Model
 {
     protected $fillable = ['project_name', 'project_info', 'project_deadline', 'project_type', 'project_status'];
```
Comment (ngon12) [inline app/Project.php]: Format this too. Thanks

**Pull request merged**